### PR TITLE
:tada: Polkadot Asset Hub 

### PR DESCRIPTION
**Pull request by vikiival**
- :wrench: speck v2
- :sparkles: enable calls for stt
- :wrench: enable stt for chain symbol
- :bug: use correct decimals
- :zap: general price function
- :sparkles: another balance format
- :zap: always correct usdPrice

**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [x] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Payout

[Payout](https://canary.kodadot.xyz/dot/transfer?target=14BZFbYEGoWWPjbbZUdLZ1TqDtdXvdeCy3R3t4QXqJmS91Dx)

## Context

- [ ] Closes #<issue_number>
- [x] Requires deployment - #kodadot/workers/130
- [ ] Related - https://github.com/kodadot/nft-gallery/issues/6094

### CODE REVIEW

**Comment by yangwao** [comment]
> Kudos, SonarCloud Quality Gate passed!

lucky

**Comment by vikiival** [comment]
Seems like minting is correctly done (kudos to @daiagi)
And price calculation too.

Screenshot 2023-08-07 at 13 57 39
Screenshot 2023-08-07 at 13 56 18
Screenshot 2023-08-07 at 13 55 54
Screenshot 2023-08-07 at 13 54 07

File: pages/stt/create.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      switchToNft,
+    }
+  },
+  head() {
+    const title = 'KodaDot | Low fees and low carbon minting'
+    const metaData = {
+      title,
+      type: 'article',
+      description: 'Create carbonless NFTs with low on-chain fees',
+      url: '/stmn/create',
```
Comment (roiLeo) [inline pages/stt/create.vue:40]:
```suggestion
      url: '/stt/create',
```

**Review by Jarsen136 — COMMENTED** [review]
small stuff

File: utils/coingecko.ts [hunk 2]
```diff
@@ -23,9 +23,40 @@ export const getPrice = async (id: string): Promise<void> => {
     return data
   } catch (error) {
     console.log(error)
+    return undefined
```
[the remark this answers is not in the document]
Comment (vikiival) [reply utils/coingecko.ts:26]: Yes this is more explicit imo + ts-lint can tell you that you return void and not undefined

**Review by Jarsen136 — APPROVED** [review]

File: components/landing/LandingPage.vue [hunk 3]
```diff
@@ -43,9 +41,13 @@
 <script lang="ts" setup>
 import type { Prefix } from '@kodadot1/static'
 
-const prefixes: Prefix[] = ['rmrk', 'ksm', 'snek', 'bsx', 'stmn']
+const prefixes: Prefix[] = ['rmrk', 'ksm', 'snek', 'bsx', 'stmn', 'stt']
+const forbiddenPrefixesForTopCollections: Prefix[] = ['ksm', 'stmn', 'stt']
```
[the remark this answers is not in the document]
Comment (vikiival) [reply components/landing/LandingPage.vue:45]: > naming personally I would have done the opposite 

And which are those? ;D

rmrk, bsx?

File: components/landing/LandingPage.vue [hunk 4]
```diff
@@ -43,9 +41,13 @@
 <script lang="ts" setup>
 import type { Prefix } from '@kodadot1/static'
 
-const prefixes: Prefix[] = ['rmrk', 'ksm', 'snek', 'bsx', 'stmn']
+const prefixes: Prefix[] = ['rmrk', 'ksm', 'snek', 'bsx', 'stmn', 'stt']
```
[the remark this answers is not in the document]
Comment (vikiival) [reply components/landing/LandingPage.vue]: 978b769be06a6526511dbab42a01f89f839d00cc

[the remark this answers is not in the document]
Comment (vikiival) [reply components/landing/LandingPage.vue]: I have also a third way

File: components/landing/LandingPage.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/landing/LandingPage.vue:45]: bsx, rmrk, snek

**Comment by prury** [comment]
Will Statemint balances be done in another PR?
how much for minting a NFT on statemint?

**Comment by vikiival** [comment]
> Will Statemint balances be done in another PR?

On the sidebar can be done right now

> how much for minting a NFT on statemint?

a lot 😅 (it is deposit)
10 DOT per collection
and 0.001 DOT per NFT ()

**Comment by yangwao** [comment]
> gosh, i will stop at the transaction screen then 😄

can reimburse some testing costs tbh 😄

**Comment by yangwao** [comment]
pay 1000 usd

**Comment by yangwao** [comment]
pay 999 usd 
i guess safe check lol

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $999 @ 5.06 USD/DOT ~ 197.431 $DOT
🧗 [14BZFbYEGoWWPjbbZUdLZ1TqDtdXvdeCy3R3t4QXqJmS91Dx](https://polkadot.subscan.io/account/14BZFbYEGoWWPjbbZUdLZ1TqDtdXvdeCy3R3t4QXqJmS91Dx)
🔗 [0xb90d4f79741a63f8d469f1bfefaa5b1b2b9f669500d2dcc5c684467b4521bf78](https://polkadot.subscan.io/extrinsic/0xb90d4f79741a63f8d469f1bfefaa5b1b2b9f669500d2dcc5c684467b4521bf78)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**